components/workflow.rst

```diff
@@ -5,8 +5,8 @@
 The Workflow Component
 ======================
 
-    The Workflow component provides tools for managing a workflow or finite state
-    machine.
+    The Workflow component provides tools for managing a workflow or finite
```

Comment:
@mickaelandrieu a workflow is a subset of a petri net (and a state machine a subset of workflow). The component does not support all features necessary to implement a petri net